src/Components/Home/Filter.js

```diff
@@ -46,9 +46,11 @@ export const Filter = props => {
                         +
                     </Link>
                 </div>
-                <button>Personal</button>
-                <button>Work</button>
-                <button>School</button>
+                <div className='filterButtonsCont'>
```

Comment:
It's exciting to see these dynamic categories instead of the previously hard coded ones!